google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbortedTest.java

```diff
@@ -115,53 +155,63 @@ public void testAbortedDuringRetryOfFailedQuery() {
 
   @Test
   public void testAbortedDuringRetryOfFailedUpdate() {
-    final Statement invalidStatement = Statement.of("INSERT INTO FOO");
+    final Iterable<Statement> invalidStatements =
+        ImmutableList.of(
+            Statement.of("INSERT INTO FOO"), Statement.of("INSERT INTO FOO THEN RETURN *"));
```

Comment:
I think it would be better to leave these tests as they were and add separate tests for DML statements with returning clauses. This potentially removes existing test coverage for DML statements without a returning clause, and those are handled differently than DML statements with a returning clause, so we should have separate tests for each.

Reply:
Added separate tests for DML statements with returning clause.